arches/app/models/resource.py

```diff
@@ -189,22 +189,31 @@ def bulk_save(resources):
             resource.tiles = resource.get_flattened_tiles()
             tiles.extend(resource.tiles)
 
-        # need to save the models first before getting the documents for index
+        # need to handle if the bulk load is appending tiles to existing resources/
+        existing_resources = Resource.objects.filter(resourceinstanceid__in=[resource.resourceinstanceid for resource in resources])
+        existing_resources_ids = [existing_resource.resourceinstanceid for existing_resource in existing_resources]
+        resources_to_create = [resource for resource in resources if resource.resourceinstanceid not in existing_resources_ids]
```

Comment:
on line 195 I would suggest as followup to my comment above:
``resources_to_create = [resource for resource in resources if resource.resourceinstanceid not in existing_resources_ids]``